### PR TITLE
169287614 dev netex documentation

### PR DESCRIPTION
**Pull request by rckopteri**
# Added
* Added new netex/chouette help file
   
# Fixed
* Search service api documentation
* log/info to log/error when there is an actual error in laundry service
   
# Changed
* Small update to main readme file

### CODE REVIEW

File: docs/api/README.md [hunk 1]
```diff
@@ -61,17 +60,27 @@ The following shows an example of the response in the JSON format:
...
       "sub-type": "terminal",
-      "id": 2
+      "id": 2,
+      "published":"1970-01-01T00:00:00Z",
+      "description":[
+        {"lang":"FI","text":"Bussipysäkki keskellä suomea"}, 
+        {"lang":"EN","text":"Bus stop in the middle of nowhere"}],
+      "business-id":"1234567-0",
+      "external-interface-links":[
+      {"external-interface":
```
Comment (philosotractor) [inline docs/api/README.md]: :url-ote-netex is not mentioned here
but in the other hand it's not in schema either:
https://testi.finap.fi/export/geojson/transport-service.schema.json

But it _is_ in geoJson export, e.g. https://testi.finap.fi/export/geojson/497/1691
I guess we can overlook this for now if you like.

File: docs/devsetup-netex.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+### How to setup netex conversion in development environment
```
Comment (philosotractor) [inline docs/devsetup-netex.md]: Ok, but if you like we could mention the netex unit test as well in section 3

File: docs/api/README.md (hunk 1 above)
Comment (rckopteri) [reply docs/api/README.md]: 👍

File: docs/devsetup-netex.md (hunk 2 above)
Comment (rckopteri) [reply docs/devsetup-netex.md]: 👍

**Pull request merged**